template.yml

```diff
@@ -328,6 +330,309 @@ Resources:
+          MONGO_URI: !Ref MongoUri
+          BIXI_DB_NAME: !Ref BixiDbName
+          BIXI_HISTORIC_URLS_COLLECTION: !Ref BixiHistoricUrlsCollection
+
+  BixiHistoricDataCheckerRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+
+  BixiHistoricDataCheckerPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: BixiHistoricDataCheckerPolicy
+      Roles:
+        - !Ref BixiHistoricDataCheckerRole
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action: s3:PutObject
+            Resource: arn:aws:s3:::monitoring-mtl-bixi-gtfs-station-status-dev/*
+          - Effect: Allow
+            Action:
+              - logs:CreateLogGroup
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+            Resource: !Join
+              - ""
+              - - !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/"
+                - !Select [
+                    1,
+                    !Split [":function:", !GetAtt BixiHistoricDataChecker.Arn],
+                  ]
+                - ":*"
+
+  BixiHistoricDataCheckerVersion:
+    Condition: SetBixiHistoricDataChecker
+    Type: AWS::Lambda::Version
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      FunctionName: !Ref BixiHistoricDataChecker
+
+  BixiHistoricDataCheckerAlias:
+    Condition: SetBixiHistoricDataChecker
+    Type: AWS::Lambda::Alias
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      FunctionName: !Ref BixiHistoricDataChecker
+      FunctionVersion: !GetAtt BixiHistoricDataCheckerVersion.Version
+      Name: Live
+
+  BixiHistoricDataCheckerPermission:
+    Type: AWS::Lambda::Permission
+    Properties:
+      Action: lambda:InvokeFunction
+      FunctionName:
+        !If [
+          SetBixiHistoricDataChecker,
+          !Ref BixiHistoricDataCheckerAlias,
+          !Join [":", [!GetAtt BixiHistoricDataChecker.Arn, "Live"]],
+        ]
+      Principal: events.amazonaws.com
+      SourceArn: !GetAtt BixiHistoricDataCheckerTrigger.Arn
+
+  ####################### BIXI - HISTORIC DATA PROCESSOR #######################
+  BixiHistoricDataProcessorLambda:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: main.handler
+      Runtime: python3.11
+      CodeUri: BIXI_Services/BIXI_Historical_Data_Processor/
+      MemorySize: 8192
+      EphemeralStorage:
+        Size: 3072
+      Timeout: 900
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - Version: "2012-10-17"
+          Statement:
+            - Effect: Allow
+              Action:
+                - logs:CreateLogStream
+                - logs:PutLogEvents
+              Resource: "arn:aws:logs:*:*:*"
```

Comment:
The section policy here is in double since you define it also at the line 477. To keep it consistent, I would remove this policy section and keep the one at 477

Reply:
Done, removed it.